src/lib/detectInstall.js

```diff
@@ -38,6 +38,12 @@ function detectInstall (context) {
   let packageName = plugin.split('@')[0]
   let packageVersion = plugin.split('@')[1] || null
 
+  let global = false
+  const basePackageName = plugin.split(path.sep).slice(-1)[0]
+  if (basePackageName=='ignite-library') {
```

Comment:
You'll want to use `===`  instead of `==`, because, well, [javascript](https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a).

Also doesn't look like you have Prettier on in your code editor. You'll want to install that and ensure it's running properly.